src/registry_testing.jl

```diff
@@ -100,6 +107,13 @@ function test(path = pwd();
                 # Test that the way Pkg loads this data works
                 Test.@test Pkg.Operations.load_package_data_raw(Pkg.Types.VersionSpec, compatfile) isa
                     Dict{Pkg.Types.VersionRange,Dict{String,Pkg.Types.VersionSpec}}
+                # Make sure the content roundtrips through decompression/compression
+                compressed = RegistryTools.Compress.compress(compatfile,
+                    RegistryTools.Compress.load(compatfile))
+                if compressed != compat
```

Comment:
```suggestion
                if compressed != compat && !(compatfile in compatfile_exceptions)
```

Reply:
And then you would define `compatfile_exceptions` as a constant somewhere in your code.